cognite/client/_cognite_client.py

```diff
@@ -52,6 +60,10 @@ def __init__(
         headers: Dict[str, str] = None,
         timeout: int = None,
         token: Union[str, Callable[[], str], None] = None,
+        token_url: str = None,
+        token_client_id: str = None,
+        token_client_secret: str = None,
+        token_scopes: List[str] = None,
```

Comment:
```suggestion
        token_url: Optional[str] = None,
        token_client_id: Optional[str] = None,
        token_client_secret: Optional[str] = None,
        token_scopes: Optional[List[str]] = None,
```

Reply:
Any reason not do this for all of these Optional arguments while I am at it?